spec/features/create_apo_spec.rb

```diff
@@ -24,5 +28,29 @@
 
     # wait for accessioningWF to finish
     reload_page_until_timeout!(text: 'v1 Accessioned', with_reindex: true)
+
+    # now register an object with this apo and verify default rights
+    visit "#{Settings.argo_url}/registration"
+    # fill in registration form
+    select apo_title, from: 'Admin Policy'
+    click_button 'Add Row'
+    td_list = all('td.invalidDisplay')
+    td_list[0].click
+    fill_in '1_source_id', with: source_id
+    td_list[1].click
+    fill_in '1_label', with: object_label
+    find_field('1_label').send_keys :enter
+
+    click_button('Register')
+    # wait for object to be registered
+    find('td[aria-describedby=data_status][title=success]')
+    object_druid = find('td[aria-describedby=data_druid]').text
+    # puts "object_druid: #{object_druid}" # useful for debugging
+
+    visit "#{Settings.argo_url}/view/#{object_druid}"
+
+    # wait for registrationWF to finish and verify default access rights
+    reload_page_until_timeout!(text: 'v1 Registered', with_reindex: true)
+    expect(page).to have_content "Access Rights:\nstanford"
```

Comment:
Can we also check copyright, license and terms of use?

Reply:
sounds reasonable, let me give that a try

Reply:
added license, terms and copyright to APO to verify they propogate to the item associated with the APO, but oddly enough i'm not seeing the license code in the cocina model of the item at all, which this test is looking for

Reply:
We don't seem to be mapping the license into cocina here: https://github.com/sul-dlss/dor-services-app/blob/main/app/services/cocina/from_fedora/access.rb#L18-L27